Project.toml

```
@@ -1,6 +1,6 @@
 name = "StaticArrays"
 uuid = "90137ffa-7385-5640-81b9-e52037218182"
-version = "1.9.8"
+version = "1.9.9"
```

```suggestion
version = "1.9.11"
```
We're already at 1.9.10.